spec/factories/languages.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :language do
+    code  { Faker::Lorem.words(3) }
```

Comment:
@haiphantony 
I think you should use `Faker::Address.country_code #=> "IT"`

https://github.com/stympy/faker/blob/master/doc/address.md